storm-core/src/clj/backtype/storm/daemon/worker.clj

```diff
@@ -335,16 +333,14 @@
         drainer (TransferDrainer.)
         node+port->socket (:cached-node+port->socket worker)
```

Comment:
There is a write-lock around `:cached-node+port->socket`, but we do not use it here in this function after binding it.